src/model/modifier/AtomicBlockUtils.js

```diff
@@ -84,6 +88,84 @@ const AtomicBlockUtils = {
 
     return EditorState.push(editorState, newContent, 'insert-fragment');
   },
+
+  moveAtomicBlock: function(
+    editorState: EditorState,
+    atomicBlock: ContentBlock,
+    targetRange: SelectionState,
+    insertionMode?: DraftInsertionType
+  ): EditorState {
+    const contentState = editorState.getCurrentContent();
+    const selectionState = editorState.getSelection();
+
+    var withMovedAtomicBlock;
+
+    if (insertionMode === 'before' || insertionMode === 'after') {
+      const targetBlock = contentState.getBlockForKey(
+        insertionMode === 'before' ?
+          targetRange.getAnchorKey() :
+          targetRange.getFocusKey()
```

Comment:
Use `getStartKey()` and `getEndKey()` for these instead, since we're looking at the before/after order of blocks irrespective of selection direction.